src/renderer/components/Sidebar.vue

```diff
@@ -1,8 +1,9 @@
 <template>
-  <section id="sidebar" :class="{'collapsed': toggleable && collapsed, 'expanded': toggleable && !collapsed}">
+  <section id="sidebar" @click="collapsed ? setCollapsed(!collapsed) : null" 
+    :class="{'collapsed': toggleable && collapsed, 'expanded': toggleable && !collapsed}">
     <aph-icon class="toggle" 
       v-if="toggleable"
-      @click.native="setCollapsed(!collapsed)" 
+      @click.stop.prevent.native="setCollapsed(!collapsed)" 
```

Comment:
Why `.native`?

Reply:
without the native the event never fires, since it on a component tag. For whatever reason @click on a component tag doesn't work without the 'native' flag. If you remove it, you can see the event actually never gets fired. 
 https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components